doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
> The plain text highlighting gets screwed up by it.

What plain text highlighting do you mean?

> it's going to get inconsistent very quickly

It already is, which is why I made these changes. Making concrete decisions about formatting documentation and documenting them sounds like a very good idea to me.